python/ray/tests/test_runtime_env_working_dir.py

```diff
@@ -266,6 +302,16 @@ def test_remote_package_uri(start_cluster, remote_uri, option, per_task_actor):
     passes with it.
     """
     cluster, address = start_cluster
+    dir_path, http_port = http_file_server
+
+    # NOTE(simon)
+    # Just downloading the remote https file for now because I don't know
+    # how was the package created.
+    if remote_uri == "use_http_magic":
+        with open(os.path.join(dir_path, "HEAD.zip"), "wb") as f:
+            resp = requests.get(HTTPS_PACKAGE_URI)
+            f.write(resp.content)
```

Comment:
@shrekris-anyscale can you help write a fixture that generates the zip file?

Reply:
🤦‍♂️ nevermind. i can just grab the github content.

Reply:
Nice work-around! Could you move the logic to download the package from the HTTPS link into the `http_file_server` fixture, and could you specify one port instead of a random choice between 30000 and 45000?

That way, you could add a global variable called `HTTP_PACKAGE_URI` and set it to a specific localhost URI. Then you could add `HTTP_PACKAGE_URI` to `REMOTE_URIS` and remove the `use_http_magic` conditional. This would allow `http` to be tested on all the tests that use `REMOTE_URIS` instead of just `test_remote_package_uri`.

Reply:
This would also require adding the `http_file_server` fixture to the tests that use `REMOTE_URIS`.